spec.bs

```diff
@@ -6292,6 +6302,10 @@ An <dfn>auction config</dfn> is a [=struct=] with the following [=struct/items=]
     encrypted response stored in [=auction config=]'s [=auction config/server response=] field.
     Must be null when [=auction config=]'s [=auction config/server response=] is null,
     and non-null otherwise.
+  : <dfn>aborted</dfn>
+  :: A [=boolean=] indicating whether the auction has been aborted, initially false.
+  : <dfn>abort reason</dfn>
+  :: Null or a [=string=], initially null.

```

Comment:
Wrong type here. It's not restricted to strings, it's actually 'any'.
https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/abort_static#reason

Reply:
Good catch!